### ISSUE TITLE
Error I got today

### ISSUE THREAD

**Issue by karrylarry23** [Booplicate/MAS-Submods-SubmodUpdaterPlugin#1]
Today when I opened the game, I got a notification to update this submod from version 1.2 to 1.4. When I went to the submod tab in Settings though, I got this error:

I'm sorry, but an uncaught exception occurred.

While loading <'Image' u'C:/Users/prasa/Desktop/Submods/Submod Updater Plugin/indicator_update_available.png'>:
IOError: Couldn't find file 'C:/Users/prasa/Desktop/Submods/Submod Updater Plugin/indicator_update_available.png'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:/Users/prasa/Desktop/_layout/screen_load_save.rpymc", line 35, in script
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\ast.py", line 814, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\python.py", line 1719, in py_exec_bytecode
    exec bytecode in globals, locals
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\ui.py", line 285, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\core.py", line 2526, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\core.py", line 2883, in interact_core
    self.draw_screen(root_widget, fullscreen_video, (not fullscreen_video) or video_frame_drawn)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\core.py", line 1955, in draw_screen
    renpy.config.screen_height,
  File "renpy/display/render.pyx", line 427, in renpy.display.render.render_screen (gen\renpy.display.render.c:6806)
    rv = render(root, width, height, 0, 0)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\transition.py", line 353, in render
    return render(self.new_widget, width, height, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\screen.py", line 623, in render
    child = renpy.display.render.render(self.child, w, h, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 693, in render
    surf = render(child, width, height, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 1081, in render
    st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 853, in render
    surf = render(d, rw, height - y, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 1081, in render
    st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 1497, in render
    cwidth, cheight = sizeit('c', width, height, 0, 0)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 1492, in sizeit
    rend = render(pos_d[pos], width, height, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\viewport.py", line 259, in render
    surf = renpy.display.render.render(self.child, child_width, child_height, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 895, in render
    surf = render(d, width - x, rh, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 895, in render
    surf = render(d, width - x, rh, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 853, in render
    surf = render(d, rw, height - y, cst, cat)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "renpy/display/accelerator.pyx", line 108, in renpy.display.accelerator.transform_render (gen\renpy.display.accelerator.c:2027)
    cr = render(child, widtho, heighto, st - self.child_st_base, at)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "renpy/display/accelerator.pyx", line 108, in renpy.display.accelerator.transform_render (gen\renpy.display.accelerator.c:2027)
    cr = render(child, widtho, heighto, st - self.child_st_base, at)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 318, in render
    surf = render(self.child, width, height, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\layout.py", line 1224, in render
    return renpy.display.render.render(self.child, w, h, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\image.py", line 387, in render
    return wrap_render(self.target, width, height, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\image.py", line 208, in wrap_render
    rend = render(child, w, h, st, at)
  File "renpy/display/render.pyx", line 110, in renpy.display.render.render (gen\renpy.display.render.c:3440)
    cpdef render(d, object widtho, object heighto, double st, double at):
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "renpy/display/accelerator.pyx", line 108, in renpy.display.accelerator.transform_render (gen\renpy.display.accelerator.c:2027)
    cr = render(child, widtho, heighto, st - self.child_st_base, at)
  File "renpy/display/render.pyx", line 196, in renpy.display.render.render (gen\renpy.display.render.c:2978)
    rv = d.render(widtho, heighto, st, at)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\im.py", line 473, in render
    im = cache.get(self)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\im.py", line 200, in get
    surf = image.load()
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\display\im.py", line 524, in load
    surf = renpy.display.pgrender.load_image(renpy.loader.load(self.filename), self.filename)
  File "C:\Users\prasa\Desktop\New folder\DDLC\Monika After story\DDLC-1.1.1-pc\renpy\loader.py", line 543, in load
    raise IOError("Couldn't find file '%s'." % name)
IOError: Couldn't find file 'C:/Users/prasa/Desktop/Submods/Submod Updater Plugin/indicator_update_available.png'.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.11.5

**Comment by Booplicate** [comment]
Please send a screenshot of `C:/Users/prasa/Desktop/New folder/DDLC/Monika After story/DDLC-1.1.1-pc/game/Submods/Submod Updater Plugin`.
I want to make sure it's installed in the right directory.

**Comment by Booplicate** [comment]
@karrylarry23 try to make a new shortcut for the game and launch the game via it. If that won't help, then it means the whole install is corrupted regarding files and filenames. Another solution would be make a new install:
- download a fresh copy of the game from [the official site](https://ddlc.moe/)
- download the mod from [the official site](http://monikaafterstory.com/releases.html)
- reinstall [this submod](https://github.com/Booplicate/MAS-Submods-SubmodUpdaterPlugin#installation) and try to launch the game

If this works, reinstall your spritepacks and other submods you had.

**Comment by karrylarry23** [comment]
There's something strange I noticed about this...
I'm on Windows 10 and I made a shortcut for MAS on my desktop as soon as I installed the mod. Whenever I use the shortcut to open the mod, I get the error above. 
But if I go directly into the DDLC folder and open the game from there, then everything works fine.
I don't know if this is even useful information, just thought it would be better to inform.
